Dockerfile

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) 2020 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#   Red Hat, Inc. - initial API and implementation
+
+FROM node:10.16-alpine
+
+ENV HOME=/home/theia
+# 1.7.1 has a bug in the packaged executable, v2.0.0 is still in beta -- thus we are using 1.7.0
+ENV VALE_VERSION=1.7.0
+
+RUN mkdir /projects ${HOME} && \
+    # Change permissions to let any arbitrary user
+    for f in "${HOME}" "/etc/passwd" "/projects"; do \
+      echo "Changing permissions on ${f}" && chgrp -R 0 ${f} && \
+      chmod -R g+rwX ${f}; \
+    done
+
+RUN apk --update --no-cache add wget bash && \
+    wget -qO- https://github.com/errata-ai/vale/releases/download/v${VALE_VERSION}/vale_${VALE_VERSION}_Linux_64-bit.tar.gz | tar xvz -C /usr/local/bin && \
+    chmod +x /usr/local/bin/vale
```

Comment:
we should probably remove wget (and bash ?) if it's not used anymore after install

Reply:
Do you mean: add them, download/install vale, then remove them?

Reply:
yes. But I'm unsure BTW we need them. Wget is installed by default on alpine. 
And for Bash do we need it ?

Reply:
I just checked and we don't, I left them there by accident for testing purposes.